packages/atauth/src/auth_cli.c

```diff
@@ -447,12 +480,13 @@ int retry_pkam_auth_until_success(atclient *ctx, const char *atsign, const atcli
   }
 }
 
-/** Fetches APKAM specific keys from server which has been encrypted using the current enrollment's APKAM SymmetricKey
+/** Fetches enrollment specific keys from server which has been encrypted using the current enrollment's APKAM
+ * SymmetricKey
  *
- * Note: It is assumed that the atclient instance has a valid authenticated connection
+ * Note: It is assumed that the atclient instance has a valid authenticated connection.
  */
-int get_apkam_key(char **key, const char *key_name, atclient_connection *ctx, const char *enrollment_id,
-                  const char *atsign) {
+int get_apkam_key(char *key, unsigned char *iv, const char *key_name, atclient_connection *ctx,
+                  const char *enrollment_id, const char *atsign) {
```

Comment:
Our convention for functions in our C SDK is `context`, then `src` arguments, then `dst` arguments last. So I would make the function signature:

```c
int get_apkam_key(atclient_connection *ctx, const char *enrollment_id, const char *atsign, char *key, unsigned char *iv, const char *key_name)
```